api/client/ps/custom.go

```diff
@@ -65,7 +65,9 @@ func (c *containerContext) Image() string {
 		return "<no image>"
 	}
 	if c.trunc {
-		return stringutils.Truncate(c.c.Image, 12)
+		if len(c.c.Image) == 64 {
```

Comment:
I agree with others that basing this on 64 chars is kind of odd. Either "trunc" applies to this field or it doesn't, irrespective of whether its user-defined or the image ID.  I don't think we should special case things for just the imageID.